app/assets/stylesheets/style.scss

```diff
@@ -393,3 +394,74 @@ label:hover {
   }
 }
 
+.side-nav {
+  transition: all 1s ease;
+  background-color: #F2F2F2;
```

Comment:
Color literals like `#F2F2F2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F2F2F2` should be written as `#f2f2f2`